resources/kcp/values.yaml

```diff
@@ -158,11 +158,12 @@ global:
     features: |-
       - name: "LOG_ISTIO_OPERATOR"
         value: "false"
-      - name: "DEBUG_LOGGING_FOR_SPECIFIC_OPERATIONS"
-        value: "true"
   component_reconcilers:
     enabled: false
     authentication: true
+  oathkeeper:
+    host: ory-oathkeeper-proxy.kyma-system.svc.cluster.local
+    port: 4455
```

Comment:
seems unrelated to SM platform, added this on purpose?

Reply:
fixed (mistake when conflicts were resovled)